.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "djinni-intellij-plugin"]
+	path = djinni-intellij-plugin
+	url = https://github.com/cross-language-cpp/djinni-intellij-plugin.git
+	branch = main
```

Comment:
I'v read that if you add a submodule like this `git submodule add -b main <url>`, the submodule will always point to the head of the included repository when checked out. I've not tried it out yet, I hope it works.

This way I do only need to check out the submodules during publishing and automatically get the newest changes from all repositories.

Reply:
This didn't work out as expected. See the Readme for more details 🙄